net/apfree-wifidog/files/wifidog.conf

```diff
@@ -0,0 +1,18 @@
+config wifidog
+	option	gateway_interface	'br-lan'
+	option	auth_server_hostname	'wifidog.kunteng.org.cn'
+	option	auth_server_port	8001
+	option	auth_server_path	'/wifidog/'	
+	option	check_interval		60
+	option	client_timeout		72000
+	option	httpd_max_conn		200
+	option	pool_mode			1
+	option	thread_number		5
+	option	queue_size			20
+	option	wired_passed		0
+	option	enable		1
```

Comment:
A few things to improve
 - Default config should be default-off, meaning `enabled` equals to 0 or `disabled` equals to 1 by default
 - Please eliminate the distribution of pki files.  This is really bad
 - Please add a README.md in the package subdir with instructions on how to deploy and how to use.

Reply:
@yousong thanks for your review.